packages/embed/src/utils/get-share-analytics-instance.ts

```diff
@@ -0,0 +1,28 @@
+const GA_TYPE_MESSAGE = 'ga-client-id'
+
+export const setupGoogleAnalyticsInstanceSharingFeature = (iframe: HTMLIFrameElement, embedId: string) => {
```

Comment:
Lets not have the function name longer than necessary:

```suggestion
export const setupGoogleAnalyticsInstance = (iframe: HTMLIFrameElement, embedId: string) => {
```

Also please rename the filename to `setup-google-analytics-instance.js`.